jest.config.ts

```diff
@@ -12,7 +12,7 @@ const baseConfig: ArrayElement<NonNullable<Config["projects"]>> = {
   // deliberately set to an empty array to allow including node_modules when transforming code:
   transformIgnorePatterns: [],
   modulePathIgnorePatterns: ["dist/"],
-  coveragePathIgnorePatterns: [".*.spec.ts", "dist/"],
+  coveragePathIgnorePatterns: [".*.spec.ts", "dist/", "node_modules/", ".*\.mock\..*"],
```

Comment:
I think if the mocks are part of our public API, then we should have test coverage on them. Then the question becomes: should we have a public API for mocks and how should that look like?

Reply:
In this case the mocks aren't part of the public API, but they are shared across multiple tests so they are exported from a separate module, resulting in them being part of the coverage measure. They aren't exported either from the top-level `index` nor are they exposed in a named export.